src/views/LobbyView.vue

```diff
@@ -70,6 +72,18 @@ export default {
       readying: false,
     };
   },
+  watch: {
+    opponentUsername: function(newVal, oldVal) {
+      // Play audio when player join lobby
+      if (oldVal === null && newVal !== null) {
+        this.$refs.enterLobbySound.play()
+      }
+      // Play audio when player leave lobby
+      else if (oldVal !== null && newVal === null) {
+        this.$refs.leaveLobbySound.play()
+      }
+    }
```

Comment:
Nice, this works. I think we can slightly simplify by looking only at the new value since if there is an opponent after this update (i.e. `newVal` is truthy) then an opponent just joined. As a super minor syntax convention, we favor the shorthand `somefunctionName() {}` over `someFunctionName: function() {}` since it saves some characters and the parentheses imply that it's a function definition.
```suggestion
    opponentUsername(newVal) {
      // Opponent joined
      if (newVall) {
        this.$refs.enterLobbySound.play()
      // Opponent left
      } else {
        this.$refs.leaveLobbySound.play()
      }
    }
```